src/SpendPermissionManager.sol

```diff
@@ -160,25 +162,24 @@ contract SpendPermissionManager is EIP712 {
     /// @notice SpendPermission was approved via transaction.
     ///
     /// @param hash The unique hash representing the spend permission.
-    /// @param account The smart contract account the spend permission controls.
     /// @param spendPermission Details of the spend permission.
-    event SpendPermissionApproved(bytes32 indexed hash, address indexed account, SpendPermission spendPermission);
+    event SpendPermissionApproved(bytes32 indexed hash, SpendPermission spendPermission);
 
     /// @notice SpendPermission was revoked by account.
     ///
     /// @param hash The unique hash representing the spend permission.
-    /// @param account The smart contract account the spend permission controlled.
     /// @param spendPermission Details of the spend permission.
-    event SpendPermissionRevoked(bytes32 indexed hash, address indexed account, SpendPermission spendPermission);
+    event SpendPermissionRevoked(bytes32 indexed hash, SpendPermission spendPermission);
```

Comment:
I wonder if we'll find ourselves wishing the account was indexed in the future, because although it's redundant indexing is pretty powerful and important for making sense of events efficiently?